lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -22,6 +48,16 @@
         Logger.log('edx.certificate.shared', data);
       });
     });
+
+    function shareOnFacebook(text) {
+        FB.ui({
+            method: 'feed',
+            name: text,
+            link: "${request.build_absolute_uri(reverse('cert_html_view', kwargs=dict(user_id=str(user.id),course_id=unicode(course_id))))}",
+            picture: "${request.build_absolute_uri(course_image_url)}",
+            description: "${_('Click on the link to see my certificate.')}"
```

Comment:
@asadiqbal08 Sorry, one more small edit: "Click the link to see my certificate." (delete "on")